finetune_and_inference_tutorial_2D_dataset.ipynb

```diff
@@ -370,7 +370,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.11"
+   "version": "3.10.6"
```

Comment:
the original `3.10.11` should work as well, right?

Reply:
Version restored!!